spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -69,8 +69,15 @@ def and_i_upload_the_file_attachment
 
   def and_i_enter_attachment_metadata
     click_on "Save and continue"
-    @metadata = "Ref: Unique ref"
-    fill_in "file_attachment[unique_reference]", with: "Unique ref"
+    unique_ref = "REF"
+    isbn = "9788700631625"
+    paper_number = "CP 1234"
+    @metadata = "Ref: ISBN #{isbn}, #{unique_ref}"
```

Comment:
This instance variable doesn't seem to be used. Without it I'm not sure you gain anything by defining the previous items as variables